migrations/Version202312071216.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\ltiDeliveryProvider\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\ltiDeliveryProvider\model\events\LtiAgsListener;
+use oat\oatbox\event\EventManager;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoDelivery\models\classes\execution\event\DeliveryExecutionState;
+
+class Version202312071216 extends AbstractMigration
+{
+    public function getDescription(): string
```

Comment:
I believe one migration is ok

Reply:
I've deleted this one and put all in one new migration.

Reply:
I've deleted this one and put all in one new migration.